prototype/link/scripts/retinotopy/FSLtoFS.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+#Takes in the Feat reg folder of the secondlevel data and makes it interoperatable with Freesurfer
+#To do this it first finds the registration.feat directories and then adds relevant information
+#It copies in the registration directory from the other analysis and then converts files as appropriate
+#
+# Naz Fall 2010
+# Vik 0313 adapting for sculpt01 retmap analysis
+# Cellis 0616 made for the development project
+#
+#SBATCH --output=logs/fsltofs-%j.out
+#SBATCH -p short
+#SBATCH -t 300
+#SBATCH --mem 20G
+
+set -ue
+
+source ./globals.sh
+
+SUBJECTS_DIR=analysis/freesurfer/; export SUBJECTS_DIR;
+
+if [ $# -eq 0 ]
+then
+	FreesurferFolder="recon"
+	echo "No Freesurfer folder supplied. Assuming recon"
+else
+	FreesurferFolder=$1
+fi
+
+featDir=analysis/secondlevel/registration.feat
+
+if [ ! -d ${featDir}/stats/ ]; then
+mkdir ${featDir}/stats/
+fi
+
+yes | cp $REGCONCAT_DIR/default/func2highres.nii.gz ${featDir}/stats/func2highres.nii.gz
+
+echo 'Replacing bad transformation matrices with good ones'
+yes | cp -f scripts/retinotopy/identitymatrix.mat $featDir/reg/example_func2initial_highres.mat
```

Comment:
This identity matrix is not included in the code release. Also what is meant by "good transformation matrices"?